common-app-lib/resources/public/site/cmr.css

```diff
@@ -0,0 +1,89 @@
+/* Home Pages
+ *
+ */
+
+.home.eui-hero {
+    background-size: cover;
+    background-color: #0d1f2b;
+    background-image: url('https://cdn.earthdata.nasa.gov/conduit/upload/947/LANCE_header_MODISAqua2.jpg');
```

Comment:
I wouldn't recommend referencing an image from this location. I'd venture a guess that Conduit uploads could be deleted. You can either pick an image from the EUI Image Library or use the image linked, but just host it as a part of your project assets.